### ISSUE TITLE
It will stop to compile if there is not any match files

### ISSUE THREAD

**Issue by fancyboynet** [webpack-contrib/stylelint-webpack-plugin#197]
- Operating System: window 10
- Node Version: v10.16.2
- NPM Version: 6.9.0
- webpack Version:  4.36.1
- stylelint-webpack-plugin Version: 1.1.0

### Expected Behavior

It should still compile next plugin whether match any files

### Code

```js
new StylelintPlugin({
      config: stylelintConfig,
      context: srcDir,
      files: '**/*.{scss,sass,css,less,html,vue}',
      ignorePattern: 'static',
      formatter: stylelintFormatter,
      fix: true
    }),
    new VueLoaderPlugin(),
```

### How Do We Reproduce?

Just delete any style files

**Comment by fancyboynet** [comment]
It seems like the `compiler.hooks.done` is not trigger

**Comment by fancyboynet** [comment]
I find if I set `lintDirtyModulesOnly` to be `true`, everything is ok, but I still don't know why

**Comment by fancyboynet** [comment]
I have created a pr to fix the bug, hope someone can handle this soon, thanks.

**Comment by ricardogobbosouza** [comment]
Hi @fancyboynet.
The mistake must really happen.
What we can do is throw the error if `failOnError` is enabled.

**Comment by ricardogobbosouza** [comment]
Resolved on v1.1.1
